pkg/process/checks/process.go

```diff
@@ -390,8 +378,9 @@ func formatCommand(fp *procutil.Process) *model.Command {
 }
 
 func formatIO(fp *procutil.Stats, lastIO *procutil.IOCountersStat, before time.Time) *model.IOStat {
-	// This will be nil for Mac
-	if fp.IOStat == nil {
+	if fp.IORateStat != nil {
+		return formatIORates(fp.IORateStat)
+	} else if fp.IOStat == nil { // This will be nil for Mac
```

Comment:
minor, what if we just remove the `else` and have another if statement here as the previous one already returns.

Reply:
Done!